@AH-64D Apache Longbow/Addons/fza_ah64_audioSystem/cfgFunctions.hpp

```diff
@@ -0,0 +1,22 @@
+#ifdef __A3_DEBUG__
+#define R recompile = 1
+#else
+#define R recompile = 0
+#endif
+class CfgFunctions
+{
+	class fza_ah64_project_audioSystem
+	{
+		tag = "FZA_audioSystem";
+		class functions {
+			file = "\fza_ah64_audioSystem\functions";
+			class addASEMessage {R;};
+			class addCaution {R;};
```

Comment:
How are advisory tones handled? The flight control tone is an example of an advisory tone. There should be a seperate handler for those.

Reply:
there is no reference to advisory tones in the documents, i need info on what they are, the priority/ implementation for it